test/gie/builtins.gie

```diff
@@ -2536,6 +2535,61 @@ accept  -200 -100
 expect  -0.001796358 -0.000904233
 
 
+===============================================================================
+Lambert Conformal Conic (2SP Michigan)
+	Conic, Sph&Ell
+	lat_1= and lat_2= or lat_0
+===============================================================================
+
+Should behave like normal LCC when scale is set to 1 (or not used).
+-------------------------------------------------------------------------------
+operation +proj=lccm +ellps=GRS80 +lat_1=0.5 +lat_2=2
```

Comment:
Instead of these generic test coordinates, maybe try to add some that are picked more deliberately. See https://coveralls.io/builds/19347707/source?filename=src%2FPJ_lccm.c#L43 for an overview of what lines you are not hitting at the moment.

The story behind the existing tests similar to these ones is that we auto-generated them all before a major restructure of the code in order to have some rudimentary regression tests. To be honest we didn't pick the most suitable coordinates, the only criteria here was to hit each quadrant of the coordinate system. When writing new test by hand more care should be taken in my opinion.

Reply:
Yes, makes sense. I took a look on coveralls report after seeing that the coverage has decreased and took note of the lines which were not tested (mostly corner cases). I'll add some more interesting tests.

Reply:
Done